packages/gafl-webapp-service/src/pages/licence-details/licence-to-start/licence-to-start.njk

```diff
@@ -58,7 +59,16 @@
     }) }}
 {% endset -%}
 
+{% block notificationContent %}
+
+    {% if SHOW_NOTIFICATION_BANNER %}
```

Comment:
Does this work? I'm not sure you have direct access to environment variables in a nunjucks template, I think you have to make them available in the `getData` method